gcexport.py

```diff
@@ -843,10 +855,75 @@ def logging_verbosity(verbosity):
             logging.debug('New console log level: %s', logging.getLevelName(level))
 
 
+def zipfilesindir(dst, src):
+    """
+    :param dst: full path with filename where the archive will be created
+    :param src: ARGS.directory - we will zip whatever is left in the directory
+    :return:
+
+    Let's unzip the archive file first, if it exists,  into a temp area. Now copy
+    the downloaded files into the temp area. This will allow for changed activities to
+    replace the older copy. Then we rebuild the archive. This is done to prevent duplicate
+    file when we re-zip.
+
+    """
+    # data_filename = os.path.join(directory, prefix + 'activity_' + activity_id + append_desc + '.zip')
+    logging.debug("In zipfilesindir, preparing to archive all file in " + src)
+    # sleep to allow system to finish processing file. We don't want an inuse or not found error
+    time.sleep(3)
+    dirpart = os.path.dirname(dst)
+    temparch = dirpart + "\\temparch"
```

Comment:
It doesn't look cross-platform. You should probably use `os.path.join()` instead.